buildscripts/ci/linux/setup.sh

```
@@ -249,6 +249,16 @@ if $BUILD_PIPEWIRE ; then
   echo export LD_LIBRARY_PATH="${pw_dist_dir}/lib:\${LD_LIBRARY_PATH}" >> ${ENV_FILE}
 fi
 
+######
+# Old ssl for crashpad
+##
+origin_dir=$(pwd)
+mkdir $BUILD_TOOLS/ssl1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)